packages/edit-site/src/store/actions.js

```diff
@@ -173,3 +173,33 @@ export function setNavigationPanelActiveMenu( menu ) {
 		menu,
 	};
 }
+
+/**
+ * todo
+ */
+export function openNavigationPanelToMenu( menu ) {
+	return {
+		type: 'OPEN_NAVIGATION_PANEL_TO_MENU',
+		menu,
+	};
+}
+
+/**
+ * todo
+ */
+export function setNavigationPanelOpen( isOpen ) {
+	return {
+		type: 'SET_NAVIGATION_PANEL_OPEN',
+		isOpen,
+	};
+}
+
+/**
+ * todo
+ */
+export function setInserterOpen( isOpen ) {
+	return {
+		type: 'SET_INSERTER_OPEN',
+		isOpen,
+	};
+}
```

Comment:
Let's keep the namings consistent with the `edit-post` version:

https://github.com/WordPress/gutenberg/blob/0ee98c8f4ca2ab50f8d17e448b906021c787ac47/packages/edit-post/src/store/actions.js#L288-L299

https://github.com/WordPress/gutenberg/blob/0ee98c8f4ca2ab50f8d17e448b906021c787ac47/packages/edit-post/src/store/reducer.js#L241-L253

https://github.com/WordPress/gutenberg/blob/0ee98c8f4ca2ab50f8d17e448b906021c787ac47/packages/edit-post/src/store/selectors.js#L317-L326

Reply:
👍 done